src/olympia/reviewers/utils.py

```diff
@@ -765,9 +765,13 @@ def reject_multiple_versions(self):
         # of the add-on instead of one of the versions we rejected, it will be
         # used to generate a token allowing the developer to reply, and that
         # only works with the latest version.
-        template = u'reject_multiple_versions'
-        subject = (u"Mozilla Add-ons: One or more versions of %s%s didn't "
-                   u"pass review")
+        if self.addon.status != amo.STATUS_PUBLIC:
+            template = u'reject_multiple_versions_disabled_addon'
+            subject = (u'Mozilla Add-ons: %s%s has been disabled on '
+                       u'addons.mozilla.org')
+        else:
+            template = u'reject_multiple_versions'
+            subject = u'Mozilla Add-ons: Versions disabled for %s%s'
```

Comment:
I think this subject reads oddly but that's what was specified so... 🤷‍♂️

Reply:
@eviljeff Can you tell me more? I know @diox just merged this but we can submit a change for the next round.

Reply:
I don't have a suggestion.  I just find "Mozilla Add-ons: Versions disabled for FooBaa" reads oddly - like "for" is implying a reason.  I'll leave bikeshedding over any change to you :)